core/Pos/Core/Common/Address.hs

```diff
@@ -84,9 +83,6 @@ import           Pos.Crypto.Signing (EncryptedSecretKey, PassPhrase, PublicKey,
                                      noPassEncrypt)
 import           Pos.Data.Attributes (attrData, mkAttributes)
 
-instance Bi Address => Hashable Address where
```

Comment:
These concrete constraints are bad -- you can ask for an unsatisfiable constraint (like `foo :: Monad Ziplist => ...`) and it will compile just fine, it just won't *work* anywhere.